.github/workflows/pinot_tests.yml

```diff
@@ -85,4 +85,6 @@ jobs:
         with:
           java-version: ${{ matrix.java }}
       - name: Quickstart on JDK ${{ matrix.java }}
+        env:
+          JAVA_OPTS: "-Xms2G -Dlog4j2.configurationFile=conf/quickstart-log4j2.xml"
```

Comment:
Do you think the jvm settings are not correctly picked up from the pom file?

Reply:
from the test, seems not, it still fails.